mv_avs_dir.sh

```diff
@@ -52,32 +52,48 @@ usage() {
     output "usage: ${bold}$SCRIPT${normal} <source-directory> <destination-directory>"
 }
 
-rm_if_link(){
-    [ ! -L "$1" ] || rm "$1";
-}
 
-rm_simlink() {
-    cd $1
+## Operates files
+##   - Removes current similinks
+##   - Creates new simlinks
+## @param destination-directory
+## @param delimiter
+##
+op_files() {
+    ##
+    ## Removes simlinks
+    ##   and Creates pair of {simlink, original}
+    ##
+    dir=$1
+    cd "$dir"
+    declare -A array
```

Comment:
No need `declare` because the code block creating symlinks can be moved into the first loop block.